config.guess

```diff
@@ -42,7 +42,7 @@ timestamp='2021-06-03'
 # have a pre-POSIX /bin/sh.
 
 
-me=`echo "$0" | sed -e 's,.*/,,'`
+me=$(echo "$0" | sed -e 's,.*/,,')
```

Comment:
I'm assuming that these changes were made when you used some different version of autoconf, and were not made by hand by you?

Reply:
> I'm assuming that these changes were made when you used some different version of autoconf, and were not made by hand by you?

I copied the whole file, but it looks like I copied a modified version. And I don't recall where I copied it from. Most probably from github.
I just checked this file on GNU repo and it does not have these modifciations
https://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess;hb=45e181800a6a27268a9c5d79dcc60492fef9a9a0

I will update the file from the official repo.